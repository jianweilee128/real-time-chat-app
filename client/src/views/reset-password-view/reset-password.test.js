import React from "react";
import { shallow } from "enzyme";
import ResetPasswordView from "./reset-password-view";

describe("ResetPasswordView", () => {
  let wrapper;
  let mockResetPassword;
  beforeEach(() => {
    mockResetPassword = jest.fn();
    const mockMatch = {
      params: {
        token: "abc123",
      },
    };

    const mockProps = {
      resetPasswordSuccess: false,
      match: mockMatch,
      resetPassword: mockResetPassword,
    };

    wrapper = shallow(<ResetPasswordView.WrappedComponent {...mockProps} />);
  });
  it("should render ResetPasswordView component", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
