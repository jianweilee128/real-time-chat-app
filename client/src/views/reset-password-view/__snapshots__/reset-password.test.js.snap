// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResetPasswordView should render ResetPasswordView component 1`] = `ShallowWrapper {}`;
